src/server/scripts/Northrend/Naxxramas/boss_noth.cpp

```diff
@@ -54,8 +54,10 @@ enum Talk
 
 enum Spells
 {
-    SPELL_CURSE         = 29213, // 25-man: 54835
-    SPELL_CRIPPLE       = 29212, // 25-man: 54814
+    SPELL_CURSE         = 29213,
+    SPELL_CRIPPLE       = 29212,
+    SPELL_CURSE_25      = 54835,
+    SPELL_CRIPPLE_25    = 54814,
```

Comment:
don't use the raid mode macro here and move the spellds to spelldifficulty so the core automatically picks the 25 player version.

Reply:
```sql
-- Noth the Plaguebringer
DELETE FROM `spelldifficulty_dbc` WHERE `id` IN (29212, 29213);
INSERT INTO `spelldifficulty_dbc` (`id`, `spellid0`, `spellid1`, `spellid2`, `spellid3`) VALUES
(29212, 29212, 54814, 0, 0), -- Cripple
(29213, 29213, 54835, 0, 0); -- Curse of the Plaguebringer
```

Reply:
I see the spell manager already handles the max targets piece, is it preferred to be inline @Jildor ?

Reply:
I disagree with that suggestion, spelldifficulty_dbc is incompatible with master (and wotlk classic)

Reply:
So this is the preferred way to handle this?  I have a bunch of other fixes for Naxx bosses hopefully I can finish this weekend

Reply:
Trying to unserstand why if pr is for 335 branch and spelldifficulty was the way, now we should avoid use that table? 

P.D. you could have waited to add the suggestion to limit the maximum targets for curse before fast merge xD

Reply:
Target limits are already in SpellMgr

Reply:
Yes, I saw it a moment ago, even so I don't understand what the standard is, is it deprecated in the future spelldifficulty_dbc?

And for maxtargets case if not in spellmgr, the standard is in spellmgr or spellmod in castspell?

Reply:
The way I see it yes, custom spelldifficulty_dbc rows are now deprecated
And for maxtargets, it doesn't matter

Reply:
https://github.com/TrinityCore/TrinityCore/blob/6b2fac687d05e14ae3e7b947b1193c39cc0d43c1/src/server/game/Spells/SpellMgr.cpp#L3447-L3452
So I was thinking about the max target value for 25 man, I think it should actually be 10?  Not sure what the standard is around here for proof of original mechanics, but one of the people I was running this with said it should be 10, I found this video from 13 years back mentioning it being 10, and wowpedia also says 10.

https://wowpedia.fandom.com/wiki/Noth_the_Plaguebringer
https://www.youtube.com/watch?v=zoQli9GzXW8